### PR TITLE
Tutorial video in Landing Page

### PR DESCRIPTION
**Pull request by SeanKennyNF**
image

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -85,12 +87,12 @@ export default async function Home() {
             backgroundImage: `url(${SeroMap.src})`,
           }}
         >
-          <div className="flex flex-col w-full rounded-md p-4 mb-4 min-h-1/2 h-fit text-background bg-white/90">
-            <div className="w-full h-fit mb-2 lg:mb-0">
+          <div className="grid grid-cols-4 w-full rounded-md p-4 mb-4 min-h-1/2 h-fit text-background bg-white/90">
```
Comment (SeanKennyNF) [inline app/page.tsx:90]: This obviously looks bad on smaller screens. I'll fix the responsiveness if we decide this is a direction we're interested in going in.

**Comment by HimanshuRanka** [comment]
This is the better alt. But can we move the buttons underneath the text and have the video just take up the right side of the screen instead

**Comment by SeanKennyNF** [comment]
Is this what you had in mind @HimanshuRanka?

image

**Comment by SeanKennyNF** [comment]
I think this looks worse than the one above but throwing it in as well.

image

**Comment by SeanKennyNF** [comment]
We're interested in moving the buttons under the text and making the video smaller than the text like heightwise

**Comment by harrietware** [comment]
The video tutorial is only for ArboTracker, not all of SeroTracker - could we make that clearer in the text or title if we keep the placement on the landing page?

**Comment by SeanKennyNF** [comment]
> The video tutorial is only for ArboTracker, not all of SeroTracker - could we make that clearer in the text or title if we keep the placement on the landing page?

For sure Harriet! This is the copy I could come up with but if we want something better, just let me know!

"Check our dashboards using the buttons to the right or view our short tutorial on how to use our ArboTracker dashboard!"

Talked with Himanshu about this one, I just made this a bit little bit better on mobile :)

**Comment by SeanKennyNF** [comment]
@harrietware + @HimanshuRanka 
Do you folks have any additional feedback here?
Are we comfortable moving this forwards?

**Review by HimanshuRanka — APPROVED** [review]

**Pull request merged**